views/js/portableElementRegistry/provider/localManifestProvider.js

```diff
@@ -49,6 +49,24 @@ define(['lodash', 'context', 'core/promise'], function(_, context, Promise){
     }
 
     /**
+     * Use portable element source if exists
+     *
+     * @param {Object} manifest - the manifest of the pci to be modified
+     * @returns {Object} the modified manifest
+     */
+    function useSource(manifest){
+        if(manifest.runtime && _.isArray(manifest.runtime.src)){
+            delete manifest.runtime.hook;//hook is going to be removed with the support of IMS PCI v1
+            manifest.runtime.libraries = manifest.runtime.src;
```

Comment:
is 'libraries` the replacement of `src` in PCI v1 ?

Reply:
src is basically the uncompiled code from PCI. It is is a TAO internal format. The "libraries" however represents the actual used code for PCIs. FYI, it is TAO specific implementation of PCI. The official PCI uses the modules instead. But it has been agreed that we will keep both implementation working side by side. We will keep improving our implementation and pushing new changes to IMS while supporting the official PCI standard.